specification/_global/reindex/types.ts

```diff
@@ -125,13 +125,21 @@ export class RemoteSource {
    */
   host: Host
   /**
-   * The username to use for authentication with the remote host.
+   * The username to use for authentication with the remote host (required when using basic auth).
    */
   username?: Username
   /**
-   * The password to use for authentication with the remote host.
+   * The password to use for authentication with the remote host (required when using basic auth).
    */
   password?: Password
+  /**
+   * The API key to use for authentication with the remote host (as an alternative to basic auth when the remote cluster is in Elastic Cloud).
+   * (It is not permitted to set this and also to set an Authorization header via {@link headers}.)
```

Comment:
While `@link` is [supported in JSDoc](https://jsdoc.app/tags-inline-link), I don't think it does what it does, especially in OpenAPI docs. Is this good enough?

```suggestion
   * (It is not permitted to set this and also to set an Authorization header via `headers`.)
```

Reply:
Yeah, absolutely. You're the expert! The thing being referenced is only a couple of lines above, so the reader shouldn't need the cross-reference.